src/man/sssd-simple.5.xml

```diff
@@ -122,8 +160,34 @@
             parameters for the simple provider using automated scripts.
         </para>
         <para>
-            Please note that it is an configuration error if both,
-            simple_allow_users and simple_deny_users, are defined.
+            **Configuration Rules:**
```

Comment:
My issue with this new section is it repeats several rules mentioned already above. The bullet points in the `DESCRIPTION` section are listing `rules` similar to what you have here. Perhaps we can combine them into a single section?